i18n/enu/out/completion/completionLibraryDcl.i18n.json

```diff
@@ -0,0 +1,7 @@
+{
+  "autolispext.commands.dclcompletion.primitive": "Primitive",
+  "autolispext.commands.dclcompletion.primitive.dialog": "Generates a dialog structure",
+  "autolispext.commands.dclcompletion.primitive.structure": "structural primitive",
+  "autolispext.commands.dclcompletion.primitive.string": "string structure",
+  "autolispext.commands.dclcompletion.primitive.boolean": "boolean structure"
+}
```

Comment:
I moved them from `i18n/enu/out/commands.i18n.json` into this file, otherwise their localized versions can't be loaded.
The key is the `completion/completionLibraryDcl` in file path, which is identical to the .ts source file.

Reply:
Thanks for fixing that